### PR TITLE
Misc code cleanup.

### PR DESCRIPTION
**Pull request by winder**
While inspecting the code I came across a few misc changes that seemed worth making.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /pushgateway
 /.build
+/.idea
```
Comment (beorn7) [inline .gitignore:3]: We cannot really accommodate individual IDEs in .gitignore.

**Review by beorn7 — CHANGES_REQUESTED** [review]
Thank you very much. All makes sense but let's not start to add IDE specific files to .gitignore (users of a specific IDE should put those files to their global gitignore).

File: .gitignore (hunk 1 above)
Comment (winder) [reply .gitignore]: Got it, I have reverted this part of the PR.

**Review by beorn7 — APPROVED** [review]
Thanks again. Will merge on green…

**Pull request merged**